android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt

```diff
@@ -333,10 +348,10 @@ class RNGestureHandlerButtonViewManager : ViewGroupManager<ButtonViewGroup>() {
         SoftAssertions.assertNotNull(attr)
         return when (attr) {
           SELECTABLE_ITEM_BACKGROUND -> {
-            R.attr.selectableItemBackground
+            android.R.attr.selectableItemBackground
```

Comment:
why this is needed? `android.R` seems to be a bit different from `R`

Reply:
Yes, `android.R` was used before migration to Kotlin (https://github.com/software-mansion/react-native-gesture-handler/commit/48468306b48207b545b47f976dcdb9f69e43e546#diff-e614cdf726640239cd9b5e551cfc1c50eb1d6f078b8c6884bfa4fb321ed82e4dL209), I'm not sure why it was changed then.